src/vaadin-rich-text-editor.html

```diff
@@ -112,6 +117,44 @@
 
     <div class="announcer" aria-live="polite"></div>
 
+    <vaadin-confirm-dialog
+      id="linkDialog"
+      opened="{{_linkEditing}}"
+      header="[[i18n.enterLinkUrl]]"
+      >
+      <vaadin-text-field
+        id="linkUrl"
+        placeholder="https://"
```

Comment:
I don’t know if this is the common practice, that the `https://` is used as a placeholder? Now it made me wonder why it’s there. My assumption was correct, that it would be added there by default if I didn’t provide either `http://` or `https://` myself.

Note: it’s not possible to use `email:` or `ftp://` prefixes. Not sure if that’s intentional, but it’s surprising. A link is still added, but with `about:blank` as the address. The field should show in an error message instead.

Reply:
My mistake with `email:`. The `mailto:` protocol worked fine.